actions/api/docs/asciidoc/dataobjects.adoc

```diff
@@ -0,0 +1,21 @@
+= Cheatsheets
+
+[[ActionFactoryOptions]]
+== ActionFactoryOptions
+
+
+[cols=">25%,25%,50%"]
+[frame="topbot"]
+|===
+^|Name | Type ^| Description
+|[[config]]`@config`|`Json object`|+++
+Sets <code>Action</code> configuration that is passed to Action.
++++
+|[[doAction]]`@doAction`|`String`|+++
+Sets the name of the base <code>Action</code> that will be triggered while creating current <code>Action</code>. In not set (<code>null</code>), given action will have no base actions.
```

Comment:
This is not really accurate - the `doAction` will not be _triggered_ when _creating current Action_ - but when the current Action is _called_. Would be nice to change even though not introduced in this PR :)